rpa-builder/6.0/modules/ROOT/pages/managing-user-templates.adoc

```diff
@@ -0,0 +1,85 @@
+= Managing Templates
+
+RPA Builder enables you to save Action Steps with all settings as templates that you can reuse in the project and you can also Import and Export templates.
+
+== Before you Begin
+
+Consider the following when creating a template:
+
+* Search Pattern and Pattern Select Action Steps and all combinations of Action Steps that include them, cannot be saved as a template.
+* RPA Builder removes all passwords from templates before creating them.
+
+== Create a Template
+
+You can create a template using the _Context menu_ or the _Workflow Tools_.
+
+* To create a Template from the _Context menu_:
++
+. Select the Action Step you want to save as a template.
+. Right-click it to open the context menu.
+. Click *Save As Template*.
+
+* To create a Template using the _Workflow Tools_:
++
+. Select the Action Step you want to save as a template.
+. In the Workbench Toolbar, click the *Tools* (image:wrench-icon.png[The Tools icon, 2%, 2%]) button.
+. Click *Save Selected Items as Template*.
+
+After selecting any of these options, a *Save as* dialog opens in which you continue configuring the template:
```

Comment:
The formatting in the preview is off here. See the List Continuation (aka Indented Paragraphs) in the Wiki for tips on lining things up: https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc